openfisca_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml

```
@@ -0,0 +1,11 @@
+description: Additional exemption amount for qualifying dependent children in IN.
+values:
+  2021-01-01: 1_500
+metadata:
+  unit: /1
```

Also in a few other parameters (capital USD)
```suggestion
  unit: currency-USD
```